src/Workspaces/Core/Portable/AddImports/AbstractAddImportsService.cs

```diff
@@ -170,7 +170,7 @@ private void GetContainers(SyntaxNode root, SyntaxNode contextLocation, out Synt
             var usingDirective = contextNode.GetAncestor<TUsingOrAliasSyntax>();
             if (usingDirective != null)
             {
-                contextNode = usingDirective.Parent;
+                contextNode = usingDirective.Parent!;
```

Comment:
❔ Can we not just mark the parameter as nullable?

Reply:
Doing so would imply that `null` was a valid argument to the function. That's not the case as the call to `contextNode.GetAncestor` would throw.

---
In reply to: [362652183](https://github.com/dotnet/roslyn/pull/40699#discussion_r362652183) [](ancestors = 362652183)

Reply:
Will rewrite to remove the suppression.

---
In reply to: [362656866](https://github.com/dotnet/roslyn/pull/40699#discussion_r362656866) [](ancestors = 362656866,362652183)